pkg/BUILD.bazel

```diff
@@ -1057,6 +1057,8 @@ GO_TARGETS = [
     "//pkg/cmd/roachtest/grafana:grafana",
     "//pkg/cmd/roachtest/option:option",
     "//pkg/cmd/roachtest/option:option_test",
+    "//pkg/cmd/roachtest/prometheus/gomock_reflect_1881716363:gomock_reflect_1881716363",
```

Comment:
are these unrelated changes?

Reply:
Thanks for catching this. I'm not sure why `dev generate bazel` seems to always put these in.

Reply:
I reverted those changes.